geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/AccumuloRasterBoundsPlanner.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2014 Commonwealth Computer Research, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package org.locationtech.geomesa.raster.data
+
+import com.typesafe.scalalogging.slf4j.Logging
+import org.apache.accumulo.core.client.IteratorSetting
+import org.locationtech.geomesa.core.iterators.BBOXCombiner
+
+case class AccumuloRasterBoundsPlanner(tableName: String) extends Logging {
```

Comment:
I think this can be an object and the one function can be made a 'static' one which takes the tableName as an argument.